assets/prototype/domain/eventEditor/containers/mutations/useCreateDateMutation.js

```diff
@@ -0,0 +1,28 @@
+import gql from "graphql-tag";
+import { useQuery } from "@apollo/react-hooks";
+import get from "lodash/get";
+
+const GET_DATETIMES = gql`
+	mutation createDatetime($input: String!) {
+		createDatetime(input: $input) {
+			datetime {
+				id
+				name
```

Comment:
We need to add all the fields that should update locally after mutation. It's better to use all the fields that were used in `useQuery`.